web/ui/src/Hosts/HostsController.js

```diff
@@ -80,52 +80,6 @@
             });
         };
 
```

Comment:
this dialog needs to be shared between host details and host page, so it is moved to the modal package where anyone can get to it.